notebooks/02-quality-flags/TIKEBook_Lesson2.ipynb

```diff
@@ -0,0 +1,1032 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    pg = lc[lc['time'].value &lt; 1339].to_periodogram(minimum_period = 1, maximum_period = 40)
</pre><p> </p><p>Could we add an exercise for using the full cadence of data? Is there any interesting "teachable moment" here?</p><p>Emma: at your discretion</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>added exercise, put solution in separate notebook. </p>